content.html

```diff
@@ -0,0 +1,143 @@
+		</tr>
+		<tr>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+		</tr>
+		<tr>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+		</tr>
+	</tbody>
+</table>
+<form method="post" action="#" class="form-horizontal">
+	<div class="form-group">
+		<label for="data1" class="col-sm-3 control-label">Form&#32;<code>input</code></label>
+		<div class="col-sm-9">
+			<input type="text" id="data1" name="data1" class="form-control" />
+		</div>
+	</div>
+	<div class="form-group">
+		<label for="data2" class="col-sm-3 control-label">Form&#32;<code>textarea</code></label>
+		<div class="col-sm-9">
+			<textarea id="data2" rows="3" cols="15" name="data2" class="form-control"></textarea>
+		</div>
+	</div>
+	<div class="form-group">
+		<label for="data4" class="col-sm-3 control-label">Form&#32;<code>select</code></label>
+		<div class="col-sm-9">
+			<select name="data4" id="data4" class="form-control">
+				<option value="1">Option&#32;1</option>
+				<option value="2">Option&#32;2</option>
+				<option value="3">Option&#32;3</option>
+				<option value="4">Option&#32;4</option>
+			</select>
+		</div>
+	</div>
+	<div class="form-group">
+		<div class="col-sm-offset-3 col-sm-9">
+			<button type="submit" class="btn btn-primary">Submit</button>
+		</div>
+	</div>
+</form>
+<blockquote>
+	<p>"<code>blockquote</code>"
+		Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text.
+	</p>
+</blockquote>
+<p><span>&mdash; Author</span><br> Example text</p>
+
+<h2> Default panel</h2>
+<section class="panel panel-default">
+	<header class="panel-heading">
+	 <h5 class="panel-title">...</h5>
+	</header>
+	<div class="panel-body">
+	 <p>...</p>
+	</div>
+  </section>
+<h2 id="call-to-action">Call to action button</h2>
+<p>Add the CSS class name <code>.btn-call-to-action</code> to a your button or link that define the main call for action for a page. For example the initiation button/link in a service initiation pages. More guidance are provided in the Content and IA specification.</p>
+<div class="row">
+	<div class="col-sm-6">
+		<h3>Link</h3>
+		<p class="mrgn-bttm-0"><a class="btn btn-call-to-action" href="#">[Call to action]</a></p>
+		<pre><code>&lt;p&gt;&lt;a class="btn <strong>btn-call-to-action</strong>" href="#"&gt;[Call to action]&lt;/a&gt;&lt;/p&gt;</code></pre>
+	</div>
+	<div class="col-sm-6">
+		<h3>Button</h3>
+		<button class="btn btn-call-to-action" type="button">[Call to action]</button>
+		<pre><code>&lt;button class="btn <strong>btn-call-to-action</strong>" type="button"&gt;[Call to action]&lt;/button&gt;</code></pre>
+	</div>
+</div>
```

Comment:
If this, which are very specific use case, is not used by OAG example, I suggest to remove it.